tensorflow/lite/tools/make/Makefile

```diff
@@ -134,6 +134,9 @@ endif
 ifeq ($(TARGET),rpi)
 	BUILD_WITH_NNAPI=false
 endif
+ifeq ($(TARGET),generic-aarch64)
```

Comment:
Just to be clear, do we hit this path for ARM64 Android builds?

Reply:
Doesn't seem like it. Android build doc suggests building demo with Gradle. This is specific to dev boards. Neither the script nor anything in tools/make is referenced from android builds as far as I can tell. That said, I don't have android devenv setup, so I can't check. FWIW, this aarch64 target didn't even exist until a couple of weeks ago, so I strongly suspect this is not what people use to build for Android.

Reply:
> @xinpingwang, could you also please address the same issue in `build_aarch64_lib.sh`? Here's how it could be addressed: define something like `generic-aarch64` target type and handle it similarly in the makefile. Then switch the makefile to use this target. I can test it for you if you don't have aarch64 hardware to test on.

@jdduke How do you think about this solution, two build target for `aarch64` devices:
- `aarch64` for devices that support `NNAPI`
- `generic-aarch64` for devices that do not support `NNAPI`

Reply:
I think it's fine if neither support NNAPI, in which case, do we really need another build script? Can we just use build_aarch64_lib.sh?